code/modules/scrap/oldificator.dm

```diff
@@ -124,6 +124,11 @@
 			reagents.remove_reagent(R.id,rand(0, R.volume),TRUE)
 		reagents.add_reagent("toxin", rand(0, actual_volume - reagents.total_volume))
 
+/obj/item/weapon/reagent_containers/food/snacks/make_old()
```

Comment:
Bit of an undeclared change, from what I can see

Reply:
Yes, I forgot to mention this change, I did it while reviewing the loot rework documentation for the tenth time to make sure everything was as planned by Ree, expired food is not supposed to provide sanity.